shell/platform/linux/fl_platform_plugin_test.cc

```diff
@@ -112,22 +112,32 @@ TEST(FlPlatformPluginTest, ExitApplication) {
 
   g_autoptr(FlPlatformPlugin) plugin = fl_platform_plugin_new(messenger);
   EXPECT_NE(plugin, nullptr);
-
-  g_autoptr(FlValue) args = fl_value_new_map();
-  fl_value_set_string_take(args, "type", fl_value_new_string("cancelable"));
   g_autoptr(FlJsonMethodCodec) codec = fl_json_method_codec_new();
-  g_autoptr(GBytes) message = fl_method_codec_encode_method_call(
-      FL_METHOD_CODEC(codec), "System.exitApplication", args, nullptr);
 
-  g_autoptr(FlValue) requestArgs = fl_value_new_map();
-  fl_value_set_string_take(requestArgs, "type",
+  ON_CALL(messenger, fl_binary_messenger_send_response(
```

Comment:
@robert-ancell In order to respond with a null argument, I had to add this `ON_CALL` to the test so that the mock would return a success response. Otherwise, because the error information isn't set, and it returns false by default, the test SEGVs when trying to access the error information. Is that expected? It doesn't seem to have any issues in production, but I want to make sure I'm not just putting duct tape on it.

Reply:
I don't fully understand the gtest mock code but I do notice this has `::testing::_` as the second last argument and not `SuccessResponse(null)` as cases in other tests? So I think it's going to be returning an empty response, not a dart null if that's what you expect.